tests/output/ConsoleInteractionTest.py

```diff
@@ -49,6 +49,15 @@
 STR_PROJECT_WIDE = "Project wide:"
 
 
+class DummyFileProxy:
+
+    def __init__(self, lines):
+        self.lines = lines
```

Comment:
The __lines won't be accesible. That is the purpose of using `__`. Maybe we can let fileproxy create two kinds of objects. One with a custom name and lines as input and one with just the absolute path of the file as input.